### PR TITLE
Prepare 0.17 release

### PR DESCRIPTION
**Pull request by joshuagl**
Fixes: # N/A

### Description of the changes being introduced by the pull request:

Update CHANGELOG and setup.py for an imminent 0.17 release

### Please verify and check that the pull request fulfils the following requirements:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+## securesystemslib v0.17.0
+
+### Added
+* Add `interface.import_publickeys_from_file()` convenience function (#278, #285)
+* Add `gpg.export_pubkeys()` convenience function (#277)
+* Add support to `hash` module for blake2b-256 algorithm (#283)
+
+### Changed
+* Use ecdsa as keytype for ECDSA keys to better distinguish between keytype
+  and scheme(#267)
```
Comment (lukpueh) [inline CHANGELOG.md:12]: Minor nit: 
```suggestion
  and scheme (#267)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Bump dependencies: cffi (#266), cryptography (#269, #274), colorama (#284)
```
Comment (lukpueh) [inline CHANGELOG.md:13]: Minor nit: (there where two cffi bumps, I guess it's okay too to only use the latter)
```suggestion
* Bump dependencies: cffi (#266, #273), cryptography (#269, #274), colorama (#284)
```

**Review by lukpueh — APPROVED** [review]
Many thanks, @joshuagl! 🎉    I checked that you didn't leave out any important changes (PRs) and that all PR numbers match the description in the changelog. LGTM, modulo 2 minor nits.

**Review by jku — APPROVED** [review]

**Comment by joshuagl** [comment]
> Many thanks, @joshuagl! 🎉 I checked that you didn't leave out any important changes (PRs) and that all PR numbers match the description in the changelog. LGTM, modulo 2 minor nits.

Thanks Lukas! I've force pushed with the nits you mentioned. Thanks for the eagle eyed review!

**Pull request merged**